pysat/tests/test_custom.py

```diff
@@ -301,20 +316,29 @@ def custom3(inst):
         self.add(custom3, 'add', at_pos=1)
         self.testInst.load(2009,1)
 
+class TestBasicsXarray(TestBasics):
+    def setup(self):
+        """Runs before every method to create a clean testing setup."""
+        self.testInst = pysat.Instrument('pysat', 'testing_xarray', tag='10',
+                                         clean_level='clean')
+
+    def teardown(self):
+        """Runs after every method to clean up previous testing."""
+        del self.testInst
 
 class ConstellationTestBasics(TestBasics):
     def setup(self):
-        '''Runs before every method to create a clean testing setup'''
+        """Runs before every method to create a clean testing setup"""
         insts = []
         for i in range(5):
             insts.append(pysat.Instrument('pysat','testing', tag='10', clean_level='clean'))
 
         self.testConst = pysat.Constellation(insts)
 
     def teardown(self):
-        ''' Runs after every method to clean up previous testing''' 
+        """ Runs after every method to clean up previous testing"""
         del self.testConst
 
     def add(self, function, kind='add', at_pos='end', *args, **kwargs):
-        ''' Add a function to the object's custom queue'''
-        self.testConst.data_mod(function, kind, at_pos, *args, **kwargs)
+        """ Add a function to the object's custom queue"""
+        self.testConst.data_mod(function, kind, at_pos, *args, **kwargs)
```

Comment:
Looks like I took out a carriage return here that may be needed when I resolved the merge conflicts.